### PR TITLE
Russell-L2521.MACC_IRR

### PR DESCRIPTION
**Pull request by russellhz**
#544 

Waiting on #548 

I tried to make the batch chunks. Hopefully I did them correctly (not really sure how to test them).

### CODE REVIEW

File: R/zchunk_L2521.MACC_IRR.R [hunk 1]
```diff
@@ -1,27 +1,25 @@
 #' module_emissions_L2521.MACC_IRR
 #'
-#' Briefly describe what this chunk does.
+#' Adds technologies and tech changes to MACC curves for animals and agriculture.
```
Comment (bpbond) [inline R/zchunk_L2521.MACC_IRR.R]: Either here or in details below, spell out what MACC stands for.

File: R/zchunk_L2521.MACC_IRR.R [hunk 2]
```diff
@@ -30,121 +28,80 @@ module_emissions_L2521.MACC_IRR_DISABLED <- function(command, ...) {
...
+      mutate(AgProductionTechnology = paste(AgSupplySubsector, Irr_Rfd, sep = "_")) %>%
+      select(-Irr_Rfd)
+
+    # Tech Change on Ag MACCs for all available SSPs
+    L2521.MAC_Ag_TC <- L2521.AgMAC %>%
+      select(-EPA_region) %>%
+      # Using left_join because the number of rows change - there is a value for each SSP
+      left_join(A_MACC_TechChange, by = c("mac.control" = "MAC")) %>%
+      # Split by scenario and add attributes since these are the same except for SSP name
+      split(.$scenario) %>%
```
Comment (bpbond) [inline R/zchunk_L2521.MACC_IRR.R]: I don't use `split` but may start doing so--this is slick.

File: inst/extdata/emissions/A_MACC_TechChange.csv [hunk 3]
```diff
@@ -0,0 +1 @@
...
SSP5,Natural Gas and Oil Systems,0.004
SSP5,Nitric and Adipic Acid Production,0.0015
SSP5,Wastewater,0.0215
SSP2,Livestock,0.03
SSP2,Cropland Management,0.019
SSP2,Rice Cultivation,0.021
SSP2,Coal Mining Activities,0.006
SSP2,Natural Gas and Oil Systems,0.008
SSP2,Nitric and Adipic Acid Production,0.003
SSP2,Wastewater,0.043
```
Comment (bpbond) [inline inst/extdata/emissions/A_MACC_TechChange.csv]: Add empty final line.

**Comment by bpbond** [comment]
>I tried to make the batch chunks. Hopefully I did them correctly (not really sure how to test them).

Thanks! It looked good to me.

Holding on @cahartin review or assignment.

File: R/zchunk_L2521.MACC_IRR.R (hunk 2 above)
Comment (cahartin) [inline R/zchunk_L2521.MACC_IRR.R]: :+1:

**Pull request merged**